src/SDK/MonitoringApplianceEmulator/Converters/BooleanToVisibilityConverter.cs

```diff
@@ -0,0 +1,72 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="BooleanToVisibilityConverter.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Converters
+{
+    using System;
+    using System.Globalization;
+    using System.Linq;
+    using System.Windows;
+    using System.Windows.Data;
+
+    /// <summary>
+    /// Implementation of <see cref="IValueConverter"/> for converting from a <see cref="bool"/> value to <see cref="Visibility"/>.
+    /// The default result will be <see cref="Visibility.Visible"/> for a value of <c>true</c>, and <see cref="Visibility.Collapsed"/> otherwise.
+    /// The result can be tuned by supplying the converter parameter, which should contain a string of two values of <see cref="Visibility"/> literals
+    /// and separated by comma (i.e. "Hidden,Visible") - the first of which represents the "true" result and the second the "zero" result.
+    /// </summary>
+    public class BooleanToVisibilityConverter : IValueConverter
```

Comment:
other converters have UTs :)

Reply:
removed this class - thought I needed it, but ended up not using it

---
In reply to: [231091864](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/13#discussion_r231091864) [](ancestors = 231091864)